lib/worker/src/routes/vaults.rs

```diff
@@ -0,0 +1,200 @@
+use std::str::FromStr;
```

Comment:
this file implements the routes defined in https://github.com/tablelandnetwork/basin-provider/pull/17/files#diff-a90fe29a31df1f6e92d758bc8bc43c20a09376f8105db546e532e9a65cd2b300R21-R32